apps/backend/src/libs/modules/plugins/libs/types/custom-request.ts

```diff
@@ -0,0 +1,7 @@
+import { type FastifyRequest } from "fastify";
+
+interface CustomFastifyRequest extends FastifyRequest {
```

Comment:
We need to add a user to the request globally. Add it to the `d.ts` file

Reply:
Do you mean to do smth like that:
```
import { type FastifyRequest } from "fastify";

type OriginalFastifyRequest = FastifyRequest;

declare module "fastify" {
  interface FastifyRequest extends OriginalFastifyRequest {
    user?: string;
  }
}
```

or just to rename that file?

Reply:
yes, like this in `backend/index.d.ts`